tests/0_spendingCondition.js

```diff
@@ -20,8 +20,8 @@ module.exports = async function(contracts, nodes, accounts, web3) {
   console.log("║2. Claim from SP by Bob                   ║");
   console.log("╚══════════════════════════════════════════╝");
 
-  let balanceAlice = (await nodes[0].web3.eth.getBalance(alice)) * 1;
-  let balanceSp = (await nodes[0].web3.eth.getBalance(spAddr)) * 1;
+  let balanceAlice = (await nodes[0].getBalance(alice)) * 1;
```

Comment:
`* 1` not needed here and everywhere as `Node.getBalance` already returns `Number`